lib/ansible/plugins/connection/ssh.py

```diff
@@ -347,7 +352,17 @@ def wrapped(self, *args, **kwargs):
                 if return_tuple[0] != 255:
```

Comment:
If you drop negation, it will be possible to drop `else` part:
```python
if return_tuple[0] == 255 and all((s not in return_tuple[1]) for s in NOT_SSH_ERRORS):
    raise AnsibleConnectionFailure("Failed to connect to the host via ssh: %s" % to_native(return_tuple[2]))
```

Reply:
Cool, fixed that.  I have to be careful... Every time I touch this, I want to use a for-else.    But the double break on top of the for-else makes it confusing for people reading the code ;-)

Reply:
Cool, fixed that.  I have to be careful... Every time I touch this, I want to use a for-else.    But the double break on top of the for-else makes it confusing for people reading the code ;-)

Reply:
Honestly I was about to suggest using for-else and then I looked closer :)